### ISSUE TITLE
Popup not blocked on https://www.google.com/amp/s/amp.reddit.com/*

### ISSUE THREAD

**Issue by cbwee** [uBlockOrigin/uAssets#8786]
### Prerequisites

- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- [x] I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [x] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin

### URL(s) where the issue occurs
https://www.google.com/amp/s/amp.reddit.com/r/explainlikeimfive/comments/ecrzvp/eli5_what_are_amp_pages_and_whats_bad_about_them/
(Open it in a mobile browser, or a desktop browser with mobile user agent)

### Describe the issue
1. Open Brave Browser on Android, enable Fanboy Annoyances List and uBlock Annoyances List in Brave://adblock .
2. Open this google search result in Brave Android, https://www.google.com/search?q=amp+reddit .
3. Click the second search result, which has an AMP symbol, and is an AMP page.
4. This page will be opened.
https://www.google.com/amp/s/amp.reddit.com/r/explainlikeimfive/comments/ecrzvp/eli5_what_are_amp_pages_and_whats_bad_about_them/
5. The popup is not blocked, as you can see in the screenshot.

6. Now test with Iceraven Browser, with uBlock Origin installed (also install Google Search Fixer).
7. In Iceraven Browser, enable uBlock filters - Annoyances and Fanboy's Annoyance.
8. Repeat steps 2 to 5, the popup is blocked, but the background of it is not blocked, as you can see in the second screenshot.
 

### Screenshot(s)
1. Brave Browser on Android
<img src="https://user-images.githubusercontent.com/34395942/112597639-57fe3880-8e48-11eb-9032-fdeab8b5107b.png" height="500">
2. Iceraven Browser
<img src="https://user-images.githubusercontent.com/34395942/112597989-e5da2380-8e48-11eb-8ec6-5825963da360.jpg" height="500">

### Versions

- Browser/version: Brave 1.22.69 Android / Iceraven 1.7.1
- uBlock Origin version: Brave Ad Block / uBlock Origin v1.34.0

### Settings

- In uBlock Origin in Iceraven, I have enabled Fanboy's Annoyance and uBlock filters - Annoyances.

### Notes
The popup still appears after 
https://github.com/uBlockOrigin/uAssets/pull/8690/commits/82fac4c60b3ff45d760089597fb9864ce1d4f784
and
https://github.com/easylist/easylist/commit/63d4fb28c1878048f55f62298aade1f7d5eef3f6

**Comment by uBlock-user** [comment]
Add `amp-reddit-com.cdn.ampproject.org##.upsell_banner.AppSelectorModal` report to Fanboy Annoyances List to get that fixed.
